src/open_inwoner/cms/plugins/cms_plugins/tasks.py

```diff
@@ -52,9 +63,27 @@ def render(self, context, instance, placeholder) -> dict[str, str]:
         if not request.user.is_authenticated or not (bsn := request.user.bsn):
             return context
 
-        task_objects = self.get_tasks_by_bsn(instance, user_bsn=bsn)
-
         task_dicts = []
+
+        # fetch "openstaande taken" from ZGW API's
+        zgw_taken = self.fetch_zgw_taken(user=request.user)
+        zgw_taken_dicts = [
+            TaskData(
+                task_url=taak.formulier_link,
+                status="open",
+                titel=taak.naam,
+                soort="",
+                verloopdatum="",
+                koppeling=None,
+                eigenaar="",
+                verwerker_taak_id="",
+            )
+            for taak in zgw_taken
+        ]
+        task_dicts.extend(zgw_taken_dicts)
```

Comment:
We can do it in a single iteration:

```suggestion
        task_dicts.extend(
            TaskData(
                    task_url=taak.formulier_link,
                    status="open",
                    titel=taak.naam,
                    soort="",
                    verloopdatum="",
                    koppeling=None,
                    eigenaar="",
                    verwerker_taak_id="",
                )
                for taak in zgw_taken
        )
```